webrender/src/picture.rs

```diff
@@ -242,8 +244,8 @@ impl PicturePrimitive {
                     }
                     Some(PictureCompositeMode::Filter(FilterOp::DropShadow(offset, blur_radius, _))) => {
                         let inflate_size = blur_radius * BLUR_SAMPLE_SCALE;
-                        local_content_rect.inflate(inflate_size, inflate_size)
-                                          .translate(&offset)
+                        *content_rect = local_content_rect.inflate(inflate_size, inflate_size);
+                        content_rect.translate(&offset)
```

Comment:
Note that technically you can apply the translation later, at the last moment.
This would allow you to merge those `match` arms, since you only need the `blur_radius` out of both to inflate the rect